l10n_th_fields/models/partner.py

```diff
@@ -61,11 +61,8 @@ def _check_partner_name(self):
     @api.constrains('vat', 'taxbranch')
     def _check_vat_taxbranch_unique(self):
         if self.vat or self.taxbranch:
-            count = self.search_count(
-                ['|', ('parent_id', '=', False),
-                 ('is_company', '=', True),
-                 ('vat', '=', self.vat),
-                 ('taxbranch', '=', self.taxbranch)])
+            count = self.search_count([('vat', '=', self.vat),
+                                       ('taxbranch', '=', self.taxbranch)])
```

Comment:
@yostashiro my apology, I really forgot that one. Now, I have added it back (also with some more enh on it, but it should not affect CFT). Also added depends on jasper_reports for tax report.